hack/e2e-test.sh

```diff
@@ -1,29 +1,33 @@
 export KUSTOMIZE=$PWD/bin/kustomize
 export GINKGO=$PWD/bin/ginkgo
 export KIND=$PWD/bin/kind
```

Comment:
Let's add `set -e` so that the script fails on any failure.

Reply:
updated